src/DIRAC/WorkloadManagementSystem/DB/JobDB.sql

```diff
@@ -42,6 +42,7 @@ CREATE TABLE `Jobs` (
   `JobName` VARCHAR(128) NOT NULL DEFAULT 'Unknown',
   `Owner` VARCHAR(64) NOT NULL DEFAULT 'Unknown',
   `OwnerGroup` VARCHAR(128) NOT NULL DEFAULT 'Unknown',
+  `VO` VARCHAR(32),
```

Comment:
Why not: 
```suggestion
  `VO` VARCHAR(32) NOT NULL DEFAULT 'Unknown',
```

Reply:
By not having a default value, a query trying to insert a row without the VO field will raise an `IntegrityError` in sqlalchemy, which will quickly indicate to the developer that something is wrong. Besides, now that the VO is used to get information from the config and that this field can no longer be determined by the `OwnerGroup`, having values as "Unknown" could be problematic.

Reply:
> a query trying to insert a row without the VO field will raise an IntegrityError in sqlalchemy

Would it raise an `IntegrityError` if the column has no `NOT NULL` constraint?

Reply:
Yes, you're right. I'll correct it.

Reply:
In the end I ended up adding the DEFAULT 'Unknown' because the _update method in DIRAC actually translates to an UPSERT, which causes an error because of the insert statement:

```
2023-09-27T08:34:51,221683Z Framework/WorkloadManagement/JobDB DEBUG: _update: INSERT INTO Jobs (JobID, Status) VALUES (3, "Received") ON DUPLICATE KEY UPDATE Status=VALUES(Status)
2023-09-27T08:34:51,221956Z Framework/WorkloadManagement/JobDB ERROR: _update (INSERT INTO Jobs (JobID, Status) VALUES (3, "Received") ON DUPLICATE KEY UPDATE Status=VALUES(Status)): Execution failed. 1364: Field 'VO' doesn't have a default value
```